File: JavaScript/Lv_0/홀수 vs 짝수.js
function solution(num_list) {
    var Odd = num_list.filter((v,i) => i%2 == 1).reduce((a,c) => a+=c);
    var Even = num_list.filter((v,i) => i%2 == 0).reduce((a,c) => a+=c);
    return Odd>Even ? Odd : Even;
}

File: C#/Lv_0/홀수 vs 짝수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int Odd = 0;
        int Even = 0;
        for(int i=0; i<num_list.Length; i++)
        {
            if(i%2 == 0) Even += num_list[i];
            else Odd += num_list[i];
        }
        return Math.Max(Odd, Even);
    }
}